jerry-core/ecma/operations/ecma-regexp-object.h

```diff
@@ -29,26 +29,30 @@
  * @{
  */
 
-#define RE_EXECUTE_RECURSION_LIMIT  1000  /* Limit of RegExp executor recursion depth */
-#define RE_EXECUTE_MATCH_LIMIT      10000 /* Limit of RegExp execetur matching steps */
+/**
+ * Limit of RegExp executor recursion depth
+ */
+#define RE_EXECUTE_RECURSION_LIMIT  1000
```

Comment:
@LaszloLango, thank you for explanation.

Could you, please, also provide a few examples of regular expressions with different recursion limits, and of an expression that would lead to infinite loop without the recursion limits?

Does the limit somehow restrict support of specification-defined semantics?